src/mlpack/core/util/arma_traits.hpp

```diff
@@ -111,4 +111,86 @@ struct IsVector<arma::subview_row<eT> >
 
 #endif
 
+// Get the row vector type corresponding to a given MatType.
+
+template<typename MatType>
+struct GetRowType
+{
+  typedef arma::Row<typename MatType::elem_type> type;
+};
+
+template<typename eT>
+struct GetRowType<arma::Mat<eT>>
+{
+  typedef arma::Row<eT> type;
+};
+
```

Comment:
we have already added these right ? I think it will be resolved while merging

Reply:
Yeah, I will resolve this during the merge.  It should be the same in several branches. :+1: